examples/example_oceandrift2.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
```

Comment:
Made this example script for testing, instead of a unittest, for easier control.